pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -188,6 +195,24 @@ func NewKubeGenericRuntimeManager(
 		typedVersion.RuntimeVersion,
 		typedVersion.RuntimeApiVersion)
 
+	runtimeConfig, err := kubeRuntimeManager.runtimeService.GetRuntimeConfigInfo()
```

Comment:
i wonder if this should go behind a feature gate since its only used to support that feature right now.

at one point, i was certain the kubelet waited 30s after startup for a runtime to be up before failing, but since we call the runtime to get the version info this is not any worse.  maybe my memory is foggy and the 30s was just waiting for the pleg to report healthy.  either way, since we have to call remotely to get the version here another call is no worse.